console/src/app/modules/smtp-provider/smtp-provider.component.html

```diff
@@ -143,6 +143,66 @@ <h2>
       </div>
     </mat-step>
 
+    <mat-step [editable]="true">
+      <form>
+        <ng-template matStepLabel>{{ 'SMTP.CREATE.STEPS.NEXT_STEPS' | translate }}</ng-template>
+        <cnsl-info-section *ngIf="!isActive">
+          <div class="title-row">
+            <div class="left">
+              <h2 class="title">{{ 'SMTP.CREATE.STEPS.ACTIVATE.TITLE' | translate }}</h2>
+              <div>
+                <a mat-icon-button card-actions mat-icon-button rel="noreferrer" target="_blank">
+                  <mat-icon class="next-icon">info_outline</mat-icon>
+                </a>
```

Comment:
This doesn't point anywhere, right?
Let's point it to https://zitadel.com/docs/guides/manage/console/default-settings#smtp

Same in the deactivate section

Reply:
Done :smile: